pkg/framework/aries/default.go

```diff
@@ -19,6 +19,12 @@ func (d defFramework) transportProviderFactory() api.TransportProviderFactory {
 	return transport.NewProviderFactory()
 }
 
+// didResolverProvider provides default DID resolver.
+func (d defFramework) didResolverProvider() DIDResolver {
+	// TODO - https://github.com/hyperledger/aries-framework-go/issues/83
+	return nil
```

Comment:
we should return didresolver.New with peer didmethod

Reply:
[Added](https://github.com/hyperledger/aries-framework-go/compare/f0888dc86951e726e7afc54054ddd0fba13b7705..0c744a0245f12cdc17b777fb7c6e3384f6335135#diff-da8e5280b0328cb616f0b42390a35b00R46) default DID Resolver with peer DID method.